app/pages/ResourceGuides/ResourceGuide.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { ResourceGuideIFrame, ResourceGuidesData } from './layout';
+import styles from './ResourceGuides.module.scss';
+import Footer from '../../components/ui/Footer/Footer';
+
+export const ResourceGuide = () => {
+  const { id } = useParams();
+  const [guide, setGuide] = useState(null);
+  const [loading, setLoading] = useState(true);
+  // const items = useMemo(() => Object.entries(ResourceGuidesLookup), []);
+
+  useEffect(() => {
```

Comment:
We had talked briefly on Slack about this, but I believe the `useEffect()` is unnecessary here, since everything that it's doing can just be done directly in the normal function body with no performance penalty.